LLama/Native/NativeApi.Load.cs

```diff
@@ -254,7 +254,7 @@ private static string GetMacLibraryPath(string libraryName)
         {
             GetPlatformPathParts(out _, out var os, out var fileExtension, out var libPrefix);
 
-            return $"runtimes/{os}/native/{libPrefix}{libraryName}{fileExtension}";
+            return $"llama/{os}/native/{libPrefix}{libraryName}{fileExtension}";
```

Comment:
I'm a bit curious about this. The binaries are copied to `any/any/llama/**/`, while `llama/**/` is used here. Will the runtime find this path without adding a search directory?

Reply:
As I understand it `any/any` isn't _really_ a folder, instead it just tells the system this content should be included with **any** language on **any** runtime. e.g. these are all valid declarations:
 - `any/net45/`
 - `cs/any`
 - `cs/netstandard2.1`
 - `any/any`

So the idea here is simply to tell the system that whatever language/runtime is using LLamaSharp, we want these files copied to these exact paths.